<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="graphTitleEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Artifacts graph</string>
   </property>
  </widget>
  <widget class="QLabel" name="graphTitleLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Title of the artifacts graph:</string>
   </property>
  </widget>
  <widget class="QLabel" name="strata_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Strata names (oldest to youngest, comma-separated):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="strataEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>XB, XA, IXB, IXA, VIII, VII, VI, Late VI</string>
   </property>
  </widget>
  <widget class="QLabel" name="kings_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kings names (oldest to youngest, comma-separated):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kingsEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sesostris I, Neferhotep I, Hatshepsut, Thutmosis III, Thutmosis IV, Amenhophis III, Ramesses I, Ramesses II, Ramesses III, Ramesses IV</string>
   </property>
  </widget>
  <widget class="QLabel" name="kingDatesLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kings earliest dates (oldest to youngest, comma-separated):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kingDatesEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-1953, -1749, -1479, -1479, -1401, -1391, -1295, -1279, -1184, -1153</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="artifactTable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>411</width>
     <height>192</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Latest  possible stratum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Earliest  possible king</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Number of artifacts</string>
    </property>
   </column>
   <item row="0" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignTrailing|AlignVCenter</set>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="addRowButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add row</string>
   </property>
  </widget>
  <widget class="QLabel" name="graphTitleLabel_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Table of artifacts:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setKingsButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setKingsDatesButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setStrataButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="graphTitleLabel_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>590</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Artifacts graph filename (JPG, TIF, PNG, PDF or SVG): </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphFilenameEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>590</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>artifacts-graph.png</string>
   </property>
  </widget>
  <widget class="QLabel" name="graphTitleLabel_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>620</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Points color:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pointsColorEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>620</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>red</string>
   </property>
  </widget>
  <widget class="QLabel" name="graphTitleLabel_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>620</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Step function color:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stepFctColorEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>620</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>red</string>
   </property>
  </widget>
  <widget class="QLabel" name="graphTitleLabel_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>620</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Figure DPI:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dpiEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>620</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>300</string>
   </property>
  </widget>
  <widget class="QLabel" name="graphTitleLabel_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>650</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X axis label:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xLabelEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>650</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Latest possible stratum</string>
   </property>
  </widget>
  <widget class="QLabel" name="graphTitleLabel_8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>650</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y axis label:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yLabelEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>650</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Earliest possible reign</string>
   </property>
  </widget>
  <widget class="QPushButton" name="makeCSVButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>790</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create TPQ table (CSV file)</string>
   </property>
  </widget>
  <widget class="QLabel" name="graphTitleLabel_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>690</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TPQ table filename (with &quot;.csv&quot; extension):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tpqFilenameEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>690</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TPQs.csv</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Please set the list of strata, kings names and kings earliest dates before filling the table of artifacts below.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>22</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Terminus Post Quem composer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="makeGraphButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>750</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create artifacts graph (PNG file)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeRowButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete row</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
